thingif/src/main/java/com/kii/thingif/command/Action.java

```diff
@@ -30,5 +24,12 @@
  *
  */
 public interface Action {
+    /**
```

Comment:
I think interface is enough, since there is not any default method. What is your reason for using abstract class?